homeassistant/components/ambient_station/__init__.py

```diff
@@ -160,6 +165,7 @@
     TYPE_BATT8: ("Battery 8", None, BINARY_SENSOR, "battery"),
     TYPE_BATT9: ("Battery 9", None, BINARY_SENSOR, "battery"),
     TYPE_BATTOUT: ("Battery", None, BINARY_SENSOR, "battery"),
+    TYPE_BATT_CO2: ("CO2 Battery", None, BINARY_SENSOR, "battery"),
```

Comment:
While you're in here, please change instances of `"battery"` to `DEVICE_CLASS_BATTERY` (make sure you import it from `homeassistant.const`).